.mergify.yml

```diff
@@ -1,4 +1,31 @@
 pull_request_rules:
+  - name: ask to fix commit message
+    conditions:
+      - -closed
+      - status-failure=Semantic Pull Request
+    actions:
+        comment:
+          message: |
+            Either the pull request or one of your commits does not follow our guidelines.
+            Consider reading https://rook.io/docs/rook/master/development-flow.html#commit-structure.
+
+            Make sure the commit "type" (its title prefix) is correct. It must be one of the following:
+                - bot
+                - build
+                - cassandra
+                - ceph
+                - ci
+                - cockroachdb
+                - core
+                - docs
+                - edgefs
+                - nfs
+                - test
+                - yugabytedb
+
+            So your commit title can be `ceph: my new feature`
```

Comment:
```suggestion
            Your commit message should follow the pattern:
            
            `<prefix>: <short 1-line description>
            
            <paragraph describing the change>`
            
            For example, if you are making a change to the Ceph storage provider:
            
            `ceph: enabling new feature x for osds
            
            Feature x for osds is needed because...`  
```